gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/java/io/gravitee/repository/jdbc/management/JdbcAuditRepository.java

```diff
@@ -96,30 +97,30 @@ public Optional<Audit> findById(String id) throws TechnicalException {
                 id
             );
             Optional<Audit> result = rowMapper.getRows().stream().findFirst();
-            LOGGER.debug("JdbcAuditRepository.findById({}) = {}", id, result);
+            log.debug("JdbcAuditRepository.findById({}) = {}", id, result);
             return result;
         } catch (final Exception ex) {
-            LOGGER.error("Failed to find audit by id:", ex);
+            log.error("Failed to find audit by id:", ex);
```

Comment:
As you update the file could you remove the log statement when we throw the exception (https://gravitee.slab.com/posts/good-practices-for-handling-java-exceptions-and-logging-1dc30qlp)